.github/workflows/benchmark.yml

```diff
@@ -52,33 +61,21 @@ jobs:
           conda info
       - name: Parse args and prepare dataset, Execute dataset
         run: |
-          IFS=',  ' read -r -a array <<< "${{ matrix.config_dataset_info }}"
-          # strip out the whitespace with the stream editor (sed)
-          CONFIG_NAME=$(echo ${array[0]} | sed 's/ //g')
-          DATASET_NAME=$(echo ${array[1]} | sed 's/ //g')
-          MAX_FRAME_LOOKAHEAD=$(echo ${array[2]} | sed 's/ //g')
-          IMAGE_EXTENSION=$(echo ${array[3]} | sed 's/ //g')
-          DATASET_SRC=$(echo ${array[4]} | sed 's/ //g')
-          LOADER_NAME=$(echo ${array[5]} | sed 's/ //g')
-          MAX_RESOLUTION=$(echo ${array[6]} | sed 's/ //g')
-          SHARE_INTRINSICS=$(echo ${array[7]} | sed 's/ //g')
-
           bash .github/scripts/execute_single_benchmark.sh \
-            $CONFIG_NAME \
-            $DATASET_NAME \
-            $MAX_FRAME_LOOKAHEAD \
-            $IMAGE_EXTENSION \
-            $DATASET_SRC \
-            $LOADER_NAME \
-            $MAX_RESOLUTION \
-            $SHARE_INTRINSICS
-
+            ${{ matrix.config_dataset_info[0] }} \ # config
+            ${{ matrix.config_dataset_info[1] }} \ # dataset
```

Comment:
otherwise I don't think the `benchmark.yaml` file is very readable because what the values mean in the matrix is not explained in the code or anywhere